### PR TITLE
Pagination adjustments

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1371195205)

### Description of Changes Made

- Only show one page either side of the current page instead of 2, to make it easier to fit the pagination in
- Adjust the mobile styles and hide some elements at mobile (previous and next text, the first and last page links)
- Add some comments to aid understanding of the template
- Display the pagination in the pattern library (note unfortunately I don't think it's possible to display it on the blog listing in the pattern library)

### How to Test
http://localhost:8000/pattern-library/pattern/patterns/styleguide/components/components.html
http://localhost:8000/pattern-library/pattern/patterns/molecules/pagination/pagination.html
http://localhost:8000/blog/

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [x] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [ ] New font variants have not been added
- [x] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

**Review by engineervix — APPROVED** [review]
Looks good to me! Good idea adding the comments in the pagination template!

File: tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html [hunk 1]
```diff
@@ -2,35 +2,41 @@
...
+                    <li class="pagination__page hidden lg:block">
                         <a href="?page=1{% if extra_url_params %}&amp;{{ extra_url_params }}{% endif %}" class="pagination__link" aria-label="Page 1 of {{ pages.paginator.num_pages }}">
                             1
                         </a>
                     </li>
-                    {% if pages.number > 4 %}
-                        <li class="pagination__page">
+                    {# Ellipsis between the first page and the page before the current page  #}
+                    {% if pages.number > 3 %}
+                        <li class="pagination__page hidden lg:block" aria-label="dot dot dot">
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:21]: Interestingly Gov UK don't add an aria-lablel to this and it's announced 'midline horizontal ellipsis'. I'm not sure what would be most useful to the user here.

Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:21]: https://design-system.service.gov.uk/components/pagination/for-list-pages/

Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:21]: Their code is `<li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>`

Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html:21]: Ah interesting - I originally copied this from RNIB, which didn't have it either, but when I tested in Safari it read it out as something like 'doooooot'. Maybe I'll remove thought to be in line with best practice.

Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html]: I've updated to use `&#8230;` for the text, but I've left the aria-label in - reason being, without it, the screen reader announces *only* where you are in terms of the list items, e.g. '3 of 7' which is confusing in the context of 'page 4 of 9' or whatever. It still reads that out afterwards, but at least this way it announces 'elipsis' first.

Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html]: Amende https://github.com/torchbox/torchbox.com/pull/53/commits/a782d823f0cf4c91d2a0c9cc92c26d36eafd0997

**Pull request merged**